docs/how_to_add_a_publisher.md

```diff
@@ -469,6 +469,24 @@ Instead, we recommend referring to [this](https://devhints.io/xpath) documentati
 Make sure to examine other parsers and consult the [attribute guidelines](attribute_guidelines.md) for specifics on attribute implementation. 
 We strongly encourage utilizing these utility functions, especially when parsing the `ArticleBody`.
 
+### Checking the free_access attribute
+
+In case your new publisher does not have a subscription model, you can go ahead and skip this step. If it does,
+please verify that there is a tag `isAccessibleForFree` within the `<script type="application/ld+json">` blocks in the
+source code of premium articles that is set to either `false` or `False`. It doesn't matter if the tag is missing in the
+freely accessible articles. If this is the case, you can continue with the next step. If not, please overwrite the
+existing function by adding the following snippet to your parser:
```

Comment:
I think in general this is a good introduction to the attribute. Just two things:
1. Rather than referring to `<script ... ld+json">` I would lead the user to the `ld` attribute of the article.
2. This section is missing in the Table of contents